bioverse/analysis.py

```diff
@@ -6,6 +6,9 @@
 import signal
 import time
 import traceback
+import logging
+
+import tqdm
```

Comment:
we can use a new package for smoother progress bars, but we'll have to add it to the requirements file.

Reply:
same for `logging`

Reply:
I don't have an issue with `tqdm`. The issue I was fixing relates to how we calculate progress, which was an issue on our end, not the package's end.